pandas/core/window.py

```diff
@@ -879,13 +879,58 @@ def f(arg, *args, **kwargs):
                            ddof=ddof, **kwargs)
 
     _shared_docs['var'] = dedent("""
-    %(name)s variance
+    Calculate unbiased %(name)s variance.
+
+    Normalized by N-1 by default. This can be changed using the ddof argument.
 
     Parameters
     ----------
     ddof : int, default 1
         Delta Degrees of Freedom.  The divisor used in calculations
-        is ``N - ddof``, where ``N`` represents the number of elements.""")
+        is ``N - ddof``, where ``N`` represents the number of elements.
+    args
+        Under Review.
+    kwargs
+        Under Review.
+
+    Returns
+    -------
+    Series or DataFrame
+        Returned object type is determined by the caller of the %(name)s
```

Comment:
It's always the same type as the original object, right? Series.rolling().var is always a Series? Let's say that.

Reply:
But DataFrame.rolling().var() is a DataFrame, so that is what the above sentences tries to say. What do you want to change? 

"is determined by " -> "same type as" ?